source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -664,7 +766,7 @@ a manner idiomatic to the Driver and Language.
      *  Connection from a closed Connection Pool
      */
     interface PoolClosedError {
-      message: 'Attempted to check out a connection from closed connection pool';
+      message: 'Attempted to check out a Connection from closed connection pool';
```

Comment:
"Connection" is capitalized in some of the code sections and lower case in others, so I just capitalized all of them for consistency. For example, the first half of the [Events](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events) section has it capitalized on master, but the second half does not (presumably different spec authors). Capitalizing them all also makes it consistent with the usage of "Endpoint" (one of the other defined terms), which is capitalized in all the code sections. Also, I wouldn't say these are like whitespace changes, since there is potential semantic meaning to differentiate here. I could go back and change them if you'd like, but I think there's value in keeping them (also it's extra work to undo them).

Reply:
I like the capitalized Connection.